hack/toolchain.sh

```diff
@@ -7,14 +7,17 @@ trap "rm -rf $TEMP_DIR" EXIT
 
 main() {
     tools
-    kubebuilder
+    if ! command -v kubebuilder &>/dev/null; then
+        install_kubebuilder
+    fi
 }
 
 tools() {
-    cd tools; GO111MODULE=on cat tools.go | grep _ | awk -F'"' '{print $2}' | xargs -tI % go install %
+    cd tools
```

Comment:
Any reason to separate these lines?

Reply:
no specific reason. I ran the script through `shfmt -i 4 -w hack/toolchain.sh` from https://github.com/mvdan/sh

It automatically split it based on some common formatting. I can move it back if you'd like

Reply:
Nope seems fine :D